### ISSUE TITLE
Some books are not scraped

### ISSUE THREAD

**Issue by aliafshany** [leoncvlt/blinkist-scraper#62]
Hello there, I hope all is well with you.
I have been using this script for a few months and I have to admit that it has significantly helped me in reading Blinkist on my kindle.
I have run into an annoying situation where I am unable to scrape some books.
For instance, I receive this error when the Chromedriver is started and the link to my book is opened.
Would you mind assisting me in completing this task?
best
![image](https://user-images.githubusercontent.com/40741745/149353229-1a674e5d-0abd-4f58-adb3-58112e112b35.png)
![image](https://user-images.githubusercontent.com/40741745/149353580-75f13bd7-72c3-46de-b13a-94fdd5805c8f.png)

**Comment by leoncvlt** [comment]
Have you tried any of the fixes suggested in #46 ?

**Comment by aliafshany** [comment]
Yes, I have tried this method and it works, but I am only able to scrape and download around half of the books I desire.

https://github.com/leoncvlt/blinkist-scraper/issues/46#issuecomment-827099491

**Comment by verebes1** [comment]
I am having the same issue; I cannot scrape any books because of the error.

**Comment by asraelvudogel** [comment]
I can confirm this. Not a single book can be scraped.
